webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue

```diff
@@ -53,6 +55,9 @@ export default {
   },
   data () {
     return {
+      cmpId: `chip${parseInt(Math.random() * 10000)
+        .toString()
+        .toString()}`,
```

Comment:
Is there a reason to have double toString() ?

Reply:
No i will fix it